frontend/src/components/pages/CamperRefundCancellation/CamperRefundFooter.tsx

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import { Flex, Button } from "@chakra-ui/react";
+
+const CamperRefundFooter = (props: any): React.ReactElement => {
+  return (
```

Comment:
Formatting issue: 
Current implementation has the button un-centered. My suggestion:
1) Change alignItems="flex-end" to alignItems="center" on the flex box
2) remove right margin (mr="50px") on the button itself